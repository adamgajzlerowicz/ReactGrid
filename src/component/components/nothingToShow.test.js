import React from 'react';
import renderer from 'react-test-renderer';

import NothingToShow from './nothingToShow';

describe('NothingToShow', () => {
  const component = renderer.create(<NothingToShow />);
  const tree = component.toJSON();

  it('renders', () => {
    expect(tree).toBeTruthy();
  });

  it('has class name', () => {
    expect(tree.props.className).toEqual('nothing-to-see');
  });

  it('has styles', () => {
    expect(tree.props.style).toBeTruthy();
  });

  it('matches the snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
