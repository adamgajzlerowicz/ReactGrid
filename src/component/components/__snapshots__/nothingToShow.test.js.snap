// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NothingToShow matches the snapshot 1`] = `
<div
  className="nothing-to-see"
  style={
    Object {
      "display": "flex",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "space-around",
      "textAlign": "center",
    }
  }
>
  There is nothing to see here
</div>
`;
